esp/src/src-react/components/ECLArchive.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import { CommandBar, ContextualMenuItemType, ICommandBarItemProps } from "@fluentui/react";
+import { WUDetails, IScope } from "@hpcc-js/comms";
+import nlsHPCC from "src/nlsHPCC";
+import { useWorkunitArchive } from "../hooks/workunit";
+import { useWorkunitMetrics } from "../hooks/metrics";
+import { HolyGrail } from "../layouts/HolyGrail";
+import { DockPanel, DockPanelItem, ResetableDockPanel } from "../layouts/DockPanel";
+import { pushUrl } from "../util/history";
+import { ShortVerticalDivider } from "./Common";
+import { ECLArchiveTree } from "./ECLArchiveTree";
+import { ECLArchiveEditor } from "./ECLArchiveEditor";
+import { MetricsPropertiesTables } from "./MetricsPropertiesTables";
+
+const scopeFilterDefault: WUDetails.RequestNS.ScopeFilter = {
+    MaxDepth: 999999,
+    ScopeTypes: []
+};
+
+const nestedFilterDefault: WUDetails.RequestNS.NestedFilter = {
+    Depth: 999999,
```

Comment:
Default should be 0?

Reply:
Not here - really it should match:  https://github.com/hpcc-systems/HPCC-Platform/blob/master/esp/src/src/ECLArchiveWidget.ts#L172-L201

I will tweak...